.github/workflows/link-check-deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Link Check Deploy
```

Comment:
I wonder if the whole workflow can be simplified- looks a bit too verbose

Reply:
I could use YAML pointers to DRY things out, and the first item isn't strictly necessary.

Reply:
Turns out GitHub Actions doesn't support YAML pointers. Where it's at now is about as simple as the workflow can be while working around GHA's weird, inconsistent treatment of Checks.